cnxepub/formatters.py

```diff
@@ -307,7 +317,7 @@ def __bytes__(self):
   </body>
 </html>
 """
-HTML_DOCUMENT = """\
+HTML_DOCUMENT = u"""\
```

Comment:
This shouldn't be necessary because we had `from __future__ import unicode_literals`.